docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -1,68 +1,72 @@
 ====================
-ClickHouse Connector
+ClickHouse connector
 ====================
 
-The ClickHouse connector allows querying tables in an external
-`Yandex ClickHouse <https://clickhouse.tech/>`_ instance. This can be used to join data between different
-systems like ClickHouse and Hive, or between two different ClickHouse instances.
+The ClickHouse connector allows querying tables in an external `Yandex
+ClickHouse <https://clickhouse.tech/>`_ instance. This can be used to join data
+between different systems like ClickHouse and Hive, or between two different
+ClickHouse instances.
 
 Configuration
 -------------
 
-To configure the ClickHouse connector, create a catalog properties file ``etc/catalog/clickhouse.properties``,
-replace the connection properties as needed for your setup:
+The connector can query a ClickHouse instance. Create a catalog properties file
+that specifies the ClickHouse connector by setting the ``connector.name`` to
+``clickhouse``.
+
+For example, to access a database as ``clickhousedb``, create the file
```

Comment:
Why would you append "db" to the catalog name?

see mysql.rst

Reply:
This is a general approach .. the catalog file name should NOT be the same as the connector name. This is a major confusion for users and we are trying to get rid of this whenever we touch a connector doc. See sqlserver for example. 

https://trino.io/docs/current/connector/sqlserver.html

Reply:
i understand, but appending `db` does not look good, because it looks like "connector for ClickHouseDB" which _could_ be a product name. 
So it can be seen as sloppy docs, even though we're trying to do the contrary.

I like the `sales` example. Maybe we use `my_catalog` here? or `my_clickhouse`?
I am not sure about others, but i like the "my xxx" convention in docs, which refers to things created by users themselves.
(obviously it doesn't work for mysql, but that can be an exception)

Reply:
I will go with myclickhouse then

Reply:
Done